packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -165,12 +170,12 @@ export class DebugSession implements Disposable {
       .then(() => disposeAll(this.disposables));
   }
 
-  public async startJSDebugSession(metro: Metro) {
+  public async startJSDebugSession(metro: Metro, extraConfiguration?: DebugExtraConfiguration) {
```

Comment:
While we're refactoring this, maybe we could remove the `metro` parameter and pass all the stuff we need in a single `Configuration` parameter?

Reply:
I don't think it is worth it at this point. Would require duplicating some of the code in multiple places or some more serious refactoring.